### ISSUE TITLE
Code minification

### ISSUE THREAD

**Issue by mdantonio** [pdonorio/restangulask#28]
I open this issue as reminder: in production we have to minify the code.

We still have to find a minifier working well with our UI stack (angular1/2/polymer/idunno).

Note: some files contain jinja templates (blueprint.js)

[UglifyJS](https://github.com/mishoo/UglifyJS2) can be a starting point to study the issue

**Comment by pdonorio** [comment]
Yes, uglify is the right way to go. And `blueprint.js` should be compiled statically for each blueprint.

The main idea is to create a `vanilla/dist` folder which will contain a minification for each blueprint. How does that sound?

**Comment by pdonorio** [comment]
Also i think this could be another task were @gianpierobozza could give us some effort, if he likes it 😄

**Comment by pdonorio** [comment]
Reading all of this about angulajs2 and gulp gave me a lot of perspective https://www.typescriptlang.org/docs/handbook/gulp.html

**Comment by mdantonio** [comment]
Waiting for the removal of flask frontend 

mdantonio/rapydo#9

Then we will be able to minify everything
